src/AppInstallerCLICore/Commands/RootCommand.cpp

```diff
@@ -66,6 +66,19 @@ namespace AppInstaller::CLI
         return "https://aka.ms/winget-command-help";
     }
 
+    void RootCommand::Execute(Execution::Context& context) const

```

Comment:
>void RootCommand::Execute(Execution::Context& context) const [](start = 4, length = 60)

We explicitly want to allow running `winget` when the whole thing is disabled?  Is that going to apply to other commands as well?

Reply:
When disabling `winget` we want to at least be able to run `winget --info` and show the policies. I don't have a strong argument for `--help` but I guess it doesn't hurt either?
We also allow `winget features` even if they are all disabled.

Reply:
I'm not a huge fan of having --info spew all of the policies, but I may not have much say.

---
In reply to: [598016873](https://github.com/microsoft/winget-cli/pull/810#discussion_r598016873) [](ancestors = 598016873)

Reply:
The policy display was an explicit request. If a user has a configuration and a policy blocks it, the user needs to know. When or if a policy is removed, the settings should again apply.